docs/blog/setup.md

```diff
@@ -1,25 +1,25 @@
-The Sysl open source setup
-======================
+The Sysl open source set-up
```

Comment:
http://www.future-perfect.co.uk/grammar-tip/is-it-setup-set-up-or-set-up/

Reply:
sublimes british en dict does not know the word setup. sorry!

Reply:
done.